### PR TITLE
report: animate between dark/light mode

### PR DESCRIPTION
**Pull request by connorjclark**
[trying more stuff from @tomayac](https://web.dev/prefers-color-scheme/).

### CODE REVIEW

File: lighthouse-core/report/html/report-styles.css [hunk 1]
```diff
@@ -825,6 +827,11 @@
   border-bottom: 1px solid var(--report-border-color-secondary);
 }
 
+/* Prevents a weird line under the budgets table. */
```
Comment (connorjclark) [inline lighthouse-core/report/html/report-styles.css]: before:
image

after:
image

**Comment by connorjclark** [comment]
> I'm not a huge fan as it stands currently though.

Me neither. I had no idea where to start with debugging the visual issues I saw - didn't know about the Animations panel. I'll make some edits and see how good we can get it.

**Comment by connorjclark** [comment]
For some reason, animating `fill` at `.lh-var` doesn't work for animating the PWA gauge.

image

Only if I put the `transition: ...` rule directly on the `circle` element does it work. Putting it on the `g` doesn't work either.

**Comment by connorjclark** [comment]
> If we can't knock them all out or more specifically knock them all out in a way without peppering transition rules all over the codebase

Yeah that's the way it seems to be going. +1 to close?

**Comment by patrickhulce** [comment]
Yeah +1 to close.

**Pull request closed without merging**